### ISSUE TITLE
SD1.5

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#62]
### Source / repo
https://huggingface.co/runwayml/stable-diffusion-v1-5 (placeholder for now)

### Model description
[DESCRIPTION]

### Dataset
[DATASET]

### Literature benchmark source
[URL]

### Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

### Strong Compute result achieved
[VALUE/S]

### Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

### Logs gist
[URL]
